runtime/Cargo.toml

```diff
@@ -37,6 +37,7 @@ solana-storage-program = { path = "../programs/storage", version = "1.1.0" }
 solana-vote-program = { path = "../programs/vote", version = "1.1.0" }
 tempfile = "3.1.0"
 thiserror = "1.0"
+thread-priority = "0.1.1"
```

Comment:
I lightly checked this crate; it seems that this is the small and nice crate for its job.

Reply:
great, thanks for looking!